### DISCUSSION TITLE
Is it possible to pass a 'x-vercel-id' header to another request?

### DISCUSSION THREAD

**Discussion by FernandoKGA** [vercel/vercel#4486] [Help] [answered]
Once I call an endpoint from Vercel, and get it's `'x-vercel-id'` header, it's possible to pass it to another request, called from inside the endpoint, to another Vercel endpoint, and use the same request id?

Will the other endpoint override the id it created with the `'x-vercel-id'` I'm passing through the headers? Or each request will never override it's `'x-vercel-id'`?

I'm asking this because I want to trace all the requests from the same parent request in order to debug errors using an integrated log app.

**Comment by styfle** [comment, marked as answer]
No, this is not possible. The `x-vercel-id` header is assigned by the platform automatically and is unique for each request.

I would suggest setting your own header than you can forward to upstream services and trace accordingly.

**Reply by FernandoKGA** [reply]
Ok, thanks @styfle !
